sensu/files/plugins/memory-metrics.rb

```diff
@@ -68,16 +68,20 @@ def metrics_hash
 
     mem['swapUsed'] = mem['swapTotal'] - mem['swapFree']
     mem['used'] = mem['total'] - mem['free']
-    memory_used = (100.0 * mem['used']) / mem['total']
+    mem['usedWOBuffersCaches'] = mem['used'] - (mem['buffers'] + mem['cached'])
+    mem['freeWOBuffersCaches'] = mem['free'] + (mem['buffers'] + mem['cached'])
+    mem['swapUsedPercentage'] = 100 * mem['swapUsed'] / mem['swapTotal'] if mem['swapTotal'] > 0
```

Comment:
Just curious, what does usedWOBuffersCaches mean? WO stands for without?

Also, just checking if this is correct, I might be wrong. Should there be + or - :-
mem['freeWOBuffersCaches'] = mem['free'] + (mem['buffers'] + mem['cached'])